api/models/reservation.py

```diff
@@ -0,0 +1,31 @@
+"""Payment model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .order import Order
+from .workshopPrice import WorkshopPrice
+
+
+class Reservation(Base):
+    """Stores workshop reservations."""
+
+    workshop_price = models.ForeignKey(
+        WorkshopPrice,
+        verbose_name=_("Workshop price"),
+    )
+    order = models.ForeignKey(
+        Order,
+        verbose_name=_("Order"),
+    )
+    ends_at = models.DateTimeField(
```

Comment:
Lower-case underscored names for model fields are convention in Django: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#model-style

We could either break the convention or translate in REST endpoints.